README.md

```diff
@@ -1,87 +1,230 @@
+## Workflows
+The Workflows folder contains examples of how more elaborate functionality is implemented by aggregating various component examples together. These examples can be used to learn how our microservices and technologies work together in a broader context. Workflow examples include:
+-	Convert a design file into a task able to be viewed on a machine control client.
+-	Stream live information about the machines connected to a site using SmartView.
+-	Configure a site with the metadata required to run the Topcon Haul App.
 
-Shell and batch scripts are provided with each component example for convenience. These supporting files include the required configuration and call a main function that runs the example code without superfluous context.
+# Running the Examples
+The steps required to run the example code will vary depending on:
+- The selected cloud environment.
+- The particular example being run.
+- The type of authorization being used.
 
-The workflows folder provides examples of more elaborate functionality achieved by chaining the component functions together. Those interested in implementing larger bodies of work such as creating a Task from a design file can browse the workflows folder for examples of how components are aggregated to achieve such outcomes. 
+The following steps describe the complete process required to run the example code from scratch:
+1.	Select a cloud environment.
+2.	Setup a Topcon account.
+3.	Obtain Service Points.
+4.	Obtain API credentials.
+5.	Configure the python environment.
+6.	Navigate to the desired example.
+7.	Configure the example.
+8.	Run the example.
 
-As with the components folder, shell and batch scripts are provided with each workflow example for convenience.
+## Select a Cloud Environment
+Sitelink3D v2 is deployed to two isolated environments:
+-	QA.
+-	Production.
 
+The choice of environment will impact the process of configuring and running example code.
 
-## Step 1. Install Python
-An installation of Python is required to run the sample scripts. At time of writing, Python 3.9.1 is the recommended version.
+### QA
+Our QA environment is a sand box intended for early testing of features before being deployed to production. QA is the recommended environment for new API users. The services deployed to QA are hosted at the following data centers:
+- "US".
 
-You can download Python 3.9.1 from:
+The QA web portal is available [here](https://qa.sitelink.topcon.com/).
+
+### Production
+Our production environment is stable and tested. Production is the recommended environment for API users who have already tested their code in QA. The services deployed to production are hosted at the following data centers:
+
+-	“US”.
+-	“EU”.
+
+The QA web portal is available [here](https://sitelink.topcon.com/).

```

Comment:
production

Reply:
Fixed, squashed and rebased onto main after previous merge this morning.